### PR TITLE
sqlite3 stub for 2.7  and 3

### PR DESCRIPTION
**Pull request by Naruto0**
I made little excursion to sqlite3 module. However, stub is incomplete and i suggest that someone with deeper knowledge of sqlite3 module build upon it. I had some issues to make sqlite3 work on python3, if this request is worth, ill make python3 version too. But not until monday.

### CODE REVIEW

File: stdlib/2.7/sqlite3/dbapi2.pyi [hunk 1]
```diff
@@ -2,14 +2,19 @@
 #
 # NOTE: This dynamically typed stub was automatically generated by stubgen.
```
Comment (gvanrossum) [inline stdlib/2.7/sqlite3/dbapi2.pyi]: This comment probably needs to go.

File: stdlib/2.7/sqlite3/dbapi2.pyi [hunk 2]
```diff
@@ -2,14 +2,19 @@
...
 
-from typing import Any
+from typing import Any, Union, List
+from numbers import Integral as _Integral
```
Comment (gvanrossum) [inline stdlib/2.7/sqlite3/dbapi2.pyi]: The "as _Integral" is actually not needed. In stubs, imported modules and classes (etc.) are not exported -- unless you use the "as blah" syntax. So this does the opposite of what you want.

File: stdlib/2.7/sqlite3/dbapi2.pyi [hunk 3]
```diff
@@ -61,18 +66,26 @@ converters = ... # type: Any
...
+# TODO: adapt needs to get probed
 def adapt(obj, protocol, alternate): ...
-def complete_statement(sql): ...
-def connect(*args, **kwargs): ...
-def enable_callback_tracebacks(flag): ...
-def enable_shared_cache(do_enable): ...
-def register_adapter(type, callable): ...
-def register_converter(typename, callable): ...
+def complete_statement(sql: str) -> bool: ...
+def connect(database: Union[bytes, unicode],\
```
Comment (gvanrossum) [inline stdlib/2.7/sqlite3/dbapi2.pyi]: The `\` is not needed because we're inside `()`.

**Comment by gvanrossum** [comment]
Thanks, I left some comments. Looking forward to your update!

**Comment by gvanrossum** [comment]
Thanks! I'll go merge this now.

**Comment by Naruto0** [comment]
Thank you very much.

**Pull request merged**